playbooks/bootstrap-software.yml

```diff
@@ -0,0 +1,12 @@
+---
+- hosts: all
+  become: true
+  tasks:
+    - name: install xfsprogs for ceph/rook
+      package:
+        name: xfsprogs
+        state: present
+    - name: install python-setuptools for ceph/rook
+      package:
+        name: python-setuptools
+        state: present
```

Comment:
present or latest?

Reply:
same with pip.  often times we get warnings when pip is too out of date.  perhaps we can have a task that ensures both `pip` and `setuptools` are up-to-date.

Reply:
I wanted to leave this as present. A few of the other playbooks (I'm thinking nvidia-dgx specifically) install a tested/validated version of python-setuptools. We don't need latest for ceph/rook to work, we just need the library present. I'd rather not upgrade/risk breaking something else if it isn't necessary.

Reply:
+1 'present' is best-practice for Ansible anyway, I think the idea is that you don't want to unknowingly update user's packages